sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -187,6 +201,12 @@ def _transmit(self, envelopes: List[TelemetryItem]) -> ExportResult:
         self._consecutive_redirects = 0
         return ExportResult.SUCCESS
 
+    # check to see whether its the case of stats collection
+    def _check_stats_collection(self):
```

Comment:
I think this function name could be better. It sounds like it's doing some sort of test. Is there something else we could call it like should_collect_stats or is_statsbeat_on?

Reply:
Replaced with `_should_collect_stats`

Reply:
Cool.